tests/test-load.js

```diff
@@ -3,13 +3,13 @@
  * @author Nicholas C. Zakas
  */
 
-"use strict";
+import assert from "node:assert"; // eslint-disable-line node/no-missing-import -- https://github.com/mysticatea/eslint-plugin-node/issues/275
 
-const assert = require("assert");
+const importFresh = async modulePath => import(`${modulePath}?x=${new Date()}`); // eslint-disable-line func-style, node/no-unsupported-features/es-syntax -- https://github.com/mysticatea/eslint-plugin-node/issues/250
```

Comment:
```suggestion
// eslint-disable-line func-style, node/no-unsupported-features/es-syntax -- https://github.com/mysticatea/eslint-plugin-node/issues/250
const importFresh = async modulePath => import(`${modulePath}?x=${new Date()}`);
```

Reply:
Updated.

Reply:
Looks like this one didn't get applied with the rest of the batch. The earlier suggestion was missing `next-`, so adding another one here for convenience:

```suggestion
// eslint-disable-next-line func-style, node/no-unsupported-features/es-syntax -- https://github.com/mysticatea/eslint-plugin-node/issues/250
const importFresh = async modulePath => import(`${modulePath}?x=${new Date()}`);
```